setup.py

```diff
@@ -539,8 +539,13 @@ def cython_extensions_and_necessity():
     author='The gRPC Authors',
     author_email='grpc-io@googlegroups.com',
     url='https://grpc.io',
+    project_urls={
+        "Source Code": "https://github.com/grpc/grpc",
+        "Bug Tracker": "https://github.com/grpc/grpc/issues",
```

Comment:
Let's also add a "Documentation" link pointing to "https://grpc.github.io/grpc/python/"